cylc/flow/simulation.py

```diff
@@ -46,23 +150,17 @@ def configure_sim_modes(taskdefs, sim_mode):
     for tdef in taskdefs:
         # Compute simulated run time by scaling the execution limit.
         rtc = tdef.rtconfig
-        sleep_sec = get_simulated_run_len(rtc)
-
-        rtc['execution time limit'] = (
-            sleep_sec + DurationParser().parse(str(
-                rtc['simulation']['time limit buffer'])).get_seconds()
-        )
 
-        rtc['simulation']['simulated run length'] = sleep_sec
         rtc['submission retry delays'] = [1]
```

Comment:
@MetRonnie  Here is the cause of the value in the GUI's submission retry delays box. I will think about whether this is required. But it's a change from master, not a fault in my branch...